definitions/private-law/json/AuthorisationCaseEvent/AuthorisationCaseEvent.json

```diff
@@ -159,6 +159,13 @@
     "UserRole": "caseworker-privatelaw-cafcass",
     "CRUD": "CRU"
   },
+  {
```

Comment:
@yogendraU Can we use `AccessControl` role definitions for this and merge all permissions into one please ?